vm/src/obj/objfunction.rs

```diff
@@ -15,6 +15,19 @@ pub fn init(context: &PyContext) {
         context.new_member_descriptor(function_code),
     );
 
+    let builtin_function_or_method_type = &context.builtin_function_or_method_type();
+    context.set_attr(
+        &builtin_function_or_method_type,
+        "__get__",
+        context.new_rustfunc(bind_method),
+    );
+
+    context.set_attr(
```

Comment:
```builtin_function_or_method_type``` doesn't have a ```__code__```. (This difference is what led me to realise we needed a different type.)

Reply:
Ok I'm working on it.

Reply:
Ok the changes are done.

Reply:
I fixed this with c04f421.